src/app/code/community/Omise/Gateway/Model/Payment/Offsiteinstalment.php

```diff
@@ -0,0 +1,161 @@
+<?php
+class Omise_Gateway_Model_Payment_Offsiteinstalment extends Omise_Gateway_Model_Payment {
+    /**
+     * @var string
+     */
+    protected $_code = 'omise_offsite_instalment';
+
+    /**
+     * @var string
+     */
+    protected $_formBlockType = 'omise_gateway/form_offsiteinstalment';
+
+    /**
+     * @var string
+     */
+    protected $_infoBlockType = 'payment/info';
+
+    /**
+     * Payment Method features
+     *
+     * @var bool
+     */
+    protected $_isGateway          = true;
+    protected $_canReviewPayment   = true;
+    protected $_isInitializeNeeded = true;
```

Comment:
probably you could consider to add comments here to each property.

Reply:
These are standard magento properties, which they don't document in their code either. The names are fairly self explanatory